agent/envoy_bootstrap/envoy_bootstrap.go

```diff
@@ -81,6 +81,18 @@ const (
 	envoyRdsRouteConf           = ".*?\\.ingress\\.\\w+?\\.[0-9]+?\\.rds\\.((.*?)\\.)(.+?)$"
 )
 
+type EnvoyCLI interface {
+	get(args ...string) (string, error)
```

Comment:
I didn't have clear favorites between get vs run. I just let it be get for `--version`. Now I changed it to say run so that we can assume the interface can be reused for other arguments.